alpha/opentracev2.cpp

```diff
@@ -54,6 +118,25 @@ void _OT_ProtocolV2::begin()
   this->bleAdvertising->setMinPreferred(0x06);  // functions that help with iPhone connections issue
   // TODO: find out what setMin and setMax interval really means
 
+  xTaskCreatePinnedToCore(
```

Comment:
no need to create new task to cleanup, can track timestamp of last cleanup and run it inline during read/write of peercache

Reply:
in fact tasks will cause concurrency issues

Reply:
run it at the end of _OT_ProtocolV2::scan_and_connect()?

Reply:
yes, or at the start. 
see how its implemented in storage.h, the cleanup function does a bunch of checks.